### DISCUSSION TITLE
Offline usage and caching

### DISCUSSION THREAD

**Discussion by RobinM30** [snakers4/silero-vad#488] [Q&A] [answered]
Hi, 
I want to load the model offline (beacause of proxys issues) but i do not succeed. I did not find the .pt file for it. I try when saving it with Colab, but i have issues with the PytorchStreamReader failed locating file constants.pkl: file not found. 
Can you help me on that?

**Comment by snakers4** [comment, marked as answer]
Hi,

> I did not find the .pt file for it

Hi, there is no `.pt`  file, the model is either in TorchScript or ONNX format [here](https://github.com/snakers4/silero-vad/tree/master/files).

One way to cache model locally is to initialize it and then just to use the created cache.

Another way would be to just cache the repo locally and then pass a local path to the saved repo to `torch.hub.load`.
